src/app/shared/interfaces/metadata.ts

```diff
@@ -0,0 +1,22 @@
+export interface MetaData {
+  checked: boolean;
+  correlationId: string;
+  duration: string;
+  endTime: string;
+  estimatedMemoryUsage: string;
+  name: string;
+  numberOfCheckpoints: string;
+  status: string;
+  storageId: string;
+  storageSize: string;
+
+  COMPONENT: string;
+  'CONVERSATION ID': string;
+  'CORRELATION ID': string;
+  'ENDPOINT NAME': string;
+  'NR OF CHECKPOINTS': string;
```

Comment:
Using this hardcoded all caps string as an attribute name is not done. Please make them camelCase and maybe use a transform function to transform the backend response with all caps: "CONVERSION ID" => conversationId into this model.

Reply:
This has been fixed